agenta-web/src/components/Layout/Layout.tsx

```diff
@@ -119,7 +119,7 @@ type LayoutProps = {
 const App: React.FC<LayoutProps> = ({children}) => {
     const {user} = useProfileData()
```

Comment:
@bekossy we need to use the loading of useProfileData as well to correctly determine loading, otherwise we'll still see this view if the profile load takes longer than expected

a better way of doing this would be to integrate loading of useProfileData into useApps loading state.